dist/sourcebuster.js

```diff
@@ -169,18 +175,38 @@ module.exports = {
     } else {
       basehost = '';
     }
-    document.cookie = this.encodeData(name) + '=' + this.encodeData(value) + expires + basehost + '; path=/';
+    var cookie_content = this.encodeData(value);
+    if (this.useBase64) {
+        // Base64 encoding and removing equal sign padding
+        cookie_content = btoa(cookie_content).replace(/=+$/, '');
+    }
+    document.cookie = this.encodeData(name) + '=' + cookie_content + expires + basehost + '; path=/';
   },
 
   get: function(name) {
     var nameEQ = this.encodeData(name) + '=',
-        ca = document.cookie.split(';');
+        ca = document.cookie.split(';'),
+        padMe = function(string) {
```

Comment:
`padMe` was just a quick example name that I think of when I was testing the function, I didn't expect you to use the name as is. 😂